### PR TITLE
Orgs using D: Update Emsi's description

### PR DESCRIPTION
**Pull request by wilzbach**
Based on http://dconf.org/2014/talks/crapuchettes.html

> An industry success story: EMSI needed to improve its data processes with n-­dimensional dataset processing, in-­memory data manipulation, data type checking, internal testing, and code correctness. D was the language of choice because it couples low-­level performance with high-­level design.

Also according to @Hackerpilot emsi is doing a rebranding to Emsi.

### CODE REVIEW

**Comment by Hackerpilot** [comment]
Ping @jcrapuchettes

**Comment by CyberShadow** [comment]
LGTM, ready to merge?

**Comment by Hackerpilot** [comment]
Not quite yet.

**Comment by wilzbach** [comment]
(rebased according to the alphabetical sorting change)

**Comment by jcrapuchettes** [comment]
LGTM

File: orgs-using-d.dd [hunk 1]
```diff
@@ -19,9 +19,13 @@ $(DIVC orgs-using-d center,
             $(FA_TESTIMONIAL) $(I $(HTTP tech.adroll.com/blog/data/2014/11/17/d-is-for-data-science.html, D is for Data Science))
          )
     )
-    $(DORG Economic modelling, http://www.economicmodeling.com, emsi.png,
-        Data-driven modelling,
-        $(FA_GITHUB economicmodeling)
+    $(DORG Emsi, http://www.economicmodeling.com, emsi.png,
+        N-dimensional dataset processing$(COMMA)
+        in-memory data manipulation
```
Comment (CyberShadow) [inline orgs-using-d.dd]: I think I think there should be a comma here

**Comment by wilzbach** [comment]
> I think I think there should be a comma here

yep :/
Fixed!

**Pull request merged**